packages/commonwealth/client/scripts/views/components/sidebar/index.tsx

```diff
@@ -18,12 +18,36 @@ import { GovernanceSection } from './governance_section';
 import { SidebarQuickSwitcher } from './sidebar_quick_switcher';
 import Permissions from '../../../utils/Permissions';
 import AccountConnectionIndicator from './AccountConnectionIndicator';
+import { Skeleton } from '../Skeleton';
 
 export type SidebarMenuName =
   | 'default'
   | 'createContent'
   | 'exploreCommunities';
 
+const SidebarSkeleton = ({ sections = 3, itemsPerSection = 5 }) => {
+  return (
+    <div className="community-menu-skeleton">
+      {Array.from({ length: sections }).map((x, index) => (
+        <div
+          className={`community-menu-skeleton-section ${
+            index > 0 ? 'mt-16' : ''
+          }`}
+        >
+          <Skeleton width={'100%'} height={25} />
+          <div className="community-menu-skeleton-section-items">
+            {Array.from({ length: itemsPerSection }).map(() => (
+              <>
+                <Skeleton width={'90%'} height={20} />
+              </>
+            ))}
+          </div>
+        </div>
+      ))}
+    </div>
+  );
+};
+
```

Comment:
I would prefer this skeleton components to be in separate files than the actual components